temba/channels/tests.py

```diff
@@ -8505,7 +8531,8 @@ def test_send(self):
             Channel.send_message(dict_to_struct('MsgStruct', msg.as_task_json()))
 
             mock.assert_called_once_with('https://devapi.globelabs.com.ph/smsmessaging/v1/outbound/21586380/requests',
-                                         headers={'User-agent': 'RapidPro'},
+                                         headers={'User-agent': 'RapidPro', 'Content-Type': 'application/json',
+                                                  'Authorization': u'key=123456789'},
```

Comment:
I think these extra headers are coming from a test leaking changes to TEMBA_HEADERS